src/components/TodoItem.tsx

```diff
@@ -0,0 +1,43 @@
+import styled from 'styled-components';
+import { ITodo } from '../interface/interface';
+import { ITodoItemTypes } from '../interface/interface';
+
+const TodoItemContainer = styled.div`
+  padding: 0.2rem 1.5rem 0.2rem 2rem;
+  display: flex;
+  justify-content: space-between;
+
+  //toggle 스타일링
+  text-decoration: ${({ isCompleted }: { isCompleted: boolean }) =>
+    isCompleted ? 'line-through' : 'none'};
+  color: ${({ isCompleted }: { isCompleted: boolean }) =>
+    isCompleted ? '#adb5bd' : 'black'};
```

Comment:
```suggestion
import styled, { css } from "styled-components";
...
${({ isCompleted }) =>
    isCompleted && {
    css`
      text-decoration: line-through;
      color: #adb5bd;
    `}
```
이렇게도 작성해볼 수 있을것 같습니다!

Reply:
wow.....props 한번만 전달 받아서 이렇게 쓸 수가 있군요.. 대박...혹시 스터디 시간에 설명 조금 부탁드려도 될까요,,,,,?